lib/beaker/host_prebuilt_steps.rb

```diff
@@ -108,6 +109,16 @@ def validate_host host, opts
     # @param [Host] host A host return the packages for
     # @return [Array<String>] A list of packages to install
     def host_packages(host)
+      packages = host_base_packages(host)
+      packages += host_timesync_packages(host) if host[:timesync]
+      packages
+    end
+
+    # Return a list of packages that should be present.
+    #
+    # @param [Host] host A host return the packages for
+    # @return [Array<String>] A list of packages to install
+    def host_base_packages(host)
```

Comment:
I am quite tempted to also simplify this, since many of them are now simply `['curl']` and there's no need to duplicate that so often.